packages/dds/tree/src/codec/fence.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
since you modified (in this case added to) the set of fence files, the diagram in the top level tree readme should be updated.